### PR TITLE
slow control integration

### PR DESCRIPTION
**Pull request by AlexWilkinsonnn**
Write run results to new epics purity monitor channels.

I added the following EPICS channels for purity monitors:
```
sbnd_prm_{inline,tpcshort,tpclong}_hv/anode_voltage
sbnd_prm_{inline,tpcshort,tpclong}_hv/cathode_voltage
sbnd_prm_{inline,tpcshort,tpclong}_hv/anodegrid_voltage
sbnd_prm_{inline,tpcshort,tpclong}_signal/QA
sbnd_prm_{inline,tpcshort,tpclong}_signal/QC
sbnd_prm_{inline,tpcshort,tpclong}_signal/drift_time
sbnd_prm_{inline,tpcshort,tpclong}_signal/lifetime
```
Currently only writing out to the HV ones, we can easily add more as needed. They are not currently linked up to the archiving system or the alarm system but we can do that easily also.

This PR requires `pyepics`, which I added to the requirements and installed on the `sbnd-prm01` system python3 also.

Let me know what you think @marcodeltutto

### CODE REVIEW

**Comment by marcodeltutto** [comment]
Hi @AlexWilkinsonnn, fantastic, thank you! I think we should go ahead and have this on the archiving system and available on CS-studio.

Also, I installed `pyepics` for all users on  `sbnd-prm01` as that requires root privileges.

**Comment by AlexWilkinsonnn** [comment]
Hi @marcodeltutto, this will be available via CS-Studio now and I can update the archiver to record these as well.

It looks like this will clash with PR #25, it will probably be easier to merge that one first and I can then pull into this branch and fix the conflicts.

File: sbndprmdaq/manager.py [hunk 1]
```diff
@@ -556,6 +562,41 @@ def save_data(self, prm_id=1):
...
+        Updates EPICS with run data.
+
+        Args:
+            prm_id (int): The purity monitor ID.
+            out_dict (dict): Data from run.
+        '''
+        if prm_id == 1:
+            prm = 'tpcshort'
+        elif prm_id == 2:
+            prm = 'tpclong'
```
Comment (marcodeltutto) [inline sbndprmdaq/manager.py:578]: Please invert long and short. I also inverted the labels on the gui in PR#25. Sorry I had them flipped to begin with

Comment (AlexWilkinsonnn) [reply sbndprmdaq/manager.py]: Ok, I swapped them

**Pull request merged**